### DISCUSSION TITLE
Mac Mouse Fix is not in applications folder [other]

### DISCUSSION THREAD

**Discussion by tonyb007** [noah-nuebling/mac-mouse-fix#191] [Questions] [answered]
I successfully installed it and am happy with how it works... but I cannot find it to adjust settings. Running Big Sur on M1 macbook air.

**Comment by noah-nuebling** [comment, marked as answer]
Hey @tonyb007, thanks for reaching out!

Maybe it's in your Downloads folder. Or you could try Spotlight to search for it (⌘ Space).

Hope that helps!

**Reply by tonyb007** [reply]
I thought I pulled this comment down because, yeah, that's all it was. Sorry - I'm a Mac newbie. App works great thank you!

**Reply by avxkim** [reply]
> Hey @tonyb007, thanks for reaching out!
> 
> Maybe it's in your Downloads folder. Or you could try Spotlight to search for it (⌘ Space).
> 
> Hope that helps!

Hey, thanks for this software! Do you have an idea, why on MacOS Ventura it does work only within ~/Downloads folder? If i move it to ~/Applications - i cannot turn the switch on.
